Detectors/MUON/MID/Raw/include/MIDRaw/RawBuffer.h

```diff
@@ -0,0 +1,64 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file   MIDRaw/RawBuffer.h
+/// \brief  Handler of the RAW buffer
+/// \author Diego Stocco <Diego.Stocco at cern.ch>
+/// \date   28 November 2019
+#ifndef O2_MID_RAWBUFFER_H
+#define O2_MID_RAWBUFFER_H
+
+#include <cstdint>
+#include <vector>
+#include <gsl/gsl>
+#include "Headers/RAWDataHeader.h"
+
+namespace o2
+{
+namespace mid
+{
+template <typename T>
+class RawBuffer
```

Comment:
This class is a bit tricky. Is there a plan to have a corresponding "testRawBuffer" to ensure the logic is sane (and does not drift in the future) ?

Reply:
The only meaningful way to test the class is when I decode raw data. So it is actually already tested at length during the test of the decoder in testRaw.cxx.